### ISSUE TITLE
Docker Hub fails building PHP < 7.0

### ISSUE THREAD

**Issue by zaporylie** [drupal-docker/php#38]
It seems to be related to PHP Redis extension. The just released  and pushed new version to PECL (3.0.0) incompatible with PHP5. I will test to build it on my localhost in a moment.

**Comment by zaporylie** [comment]
Related log from hub.docker.com:

```
Installing shared extensions:     /usr/local/lib/php/extensions/no-debug-non-zts-20131226/
find . -name \*.gcno -o -name \*.gcda | xargs rm -f
find . -name \*.lo -o -name \*.o | xargs rm -f
find . -name \*.la -o -name \*.a | xargs rm -f 
find . -name \*.so | xargs rm -f
find . -name .libs -a -type d|xargs rm -rf
rm -f libphp.la       modules/* libs/*
pecl/redis requires PHP (version >= 7.0.0, version <= 7.1.0, excluded versions: 6.0.0), installed version is 5.6.22
No valid packages found
install failed
```

**Comment by zaporylie** [comment]
Fixed #39
